system.json

```diff
@@ -2,7 +2,7 @@
   "name": "prime",
   "title": "Prime system",
   "description": "The foundry VTT system of the Prime RPG as used in the Possession world.",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
We have a new migration for 0.3.1-->0.4.0, this patch version bump to 0.3.2 isn't covered by the case statement.

Reply:
Haha, good spot, I'll sync. those up.

Reply:
Should be all good now

Reply:
Cheers!